Makefile

```diff
@@ -165,4 +165,5 @@ format:
 
 .PHONY: lint
 lint:
+	@npx stylelint --ignore-path .gitignore "**/*.scss" --syntax scss
```

Comment:
typically in `make` we'd create two new targets and use `lint` to tie them together. this makes the built-in dependency resolution work easier and gives us the ability to lint just the JS or just the SCSS

```makefile
.PHONY: lint
lint: lint-js lint-scss

.PHONY: lint-js
lint-js:
	@npx eslint --ignore-path .gitignore "**/*.{js,jsx,ts,tsx}"

.PHONY: lint-scss
lint-scss:
	@npx stylelint --ignore-path .gitignore "**/*.scss" --syntax scss
```

there are other enhancements we could make but for this PR I think this is the relevant bit. this may allow for parallelism as well